### PR TITLE
NationalSecurityAgency.Ghidra version 10.2.3

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98933)

### CODE REVIEW

File: manifests/n/NationalSecurityAgency/Ghidra/10.2.3/NationalSecurityAgency.Ghidra.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NationalSecurityAgency.Ghidra
+PackageVersion: 10.2.3
+PackageLocale: en-US
+Publisher: National Security Agency of the United States of America
+PackageName: Ghidra
```
Comment (stephengillie) [inline manifests/n/NationalSecurityAgency/Ghidra/10.2.3/NationalSecurityAgency.Ghidra.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.nsa.gov/
PrivacyUrl:   https://www.defense.gov/Legal-Administrative/Privacy-Security/
PackageName: Ghidra
PackageUrl: https://ghidra-sre.org/
```

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The installer did not complete:
image

File: manifests/n/NationalSecurityAgency/Ghidra/10.2.3/NationalSecurityAgency.Ghidra.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: NationalSecurityAgency.Ghidra
+PackageVersion: 10.2.3
+Installers:
+- InstallerUrl: https://github.com/NationalSecurityAgency/ghidra/releases/download/Ghidra_10.2.3_build/ghidra_10.2.3_PUBLIC_20230208.zip
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: ghidra_10.2.3_PUBLIC\Ghidra\Features\Python\data\jython-2.7.3\Lib\distutils\command\wininst-9.0.exe
```
Comment (chq-matteo) [inline manifests/n/NationalSecurityAgency/Ghidra/10.2.3/NationalSecurityAgency.Ghidra.installer.yaml:12]: would it make sense to drop the jython installation for a first release?

**Comment by stephengillie** [comment]
Closing as No Recent Activity.

**Pull request closed without merging**